src/components/style/button.css

```diff
@@ -227,6 +249,24 @@
 .fdsButton--outlined.fdsButton--isDestructive.fdsButton--isActive:focus {
   background-color: #fad6d8;
 }
+
+.fdsButton--outlined--blue.fdsButton--isDestructive.fdsButton--isActive {
```

Comment:
do we want think pinkish color for active state of `blue-outlined`?

Reply:
looks like this gets reset later in the cascade - do the outlined blue variant classes need to be in these selectors?

Reply:
This is for the `isDestructive` modifier, which is for all intents and purposes a separate theme / theme modifier.

<img width="284" alt="Screen Shot 2021-03-12 at 5 24 34 PM" src="https://user-images.githubusercontent.com/708820/111008916-d2fb2580-8357-11eb-8dbf-4d6c38ca3ce3.png">

We may need to rethink this button api holistically though as we discussed, but there doesn't seem to be an ideal choice, just what tradeoffs are we happiest with 😅

Reply:
I forgot it was a separate prop! Works for me